ansible_collections/arista/avd/examples/cv-pathfinder/group_vars/WAN/vault.yml

```diff
@@ -0,0 +1,8 @@
+$ANSIBLE_VAULT;1.1;AES256
```

Comment:
I think it would be more userfriendly if we used the inline vault so we can see which vars are vaulted.